runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -2948,7 +2950,7 @@ J9::ValuePropagation::isArrayElementFlattened(TR::VPConstraint *arrayConstraint)
       return TR_no;
       }
 
-   TR_YesNoMaybe isCompTypePrimVT = isArrayCompTypePrimitiveValueType(arrayConstraint);
+   TR_YesNoMaybe isCompTypePrimVT = isArrayNullRestricted(arrayConstraint);
```

Comment:
This local variable `isCompTypePrimVT` should be renamed to reflect the new concept of null-restricted arrays.